tests/unit_tests/service/test_runner.py

```diff
@@ -186,42 +186,42 @@ class GenericModel(BaseModel, Generic[T]):
     b: str
 
 
-def return_int() -> int:
+def return_int(*args, **kwargs) -> int:
```

Comment:
yuck

Reply:
It's not my stupid programming language :)

Reply:
Closing this thread in favour of [the bigger one](https://github.com/DiamondLightSource/blueapi/pull/669#discussion_r1802994755). I will write an explanation there.